### PR TITLE
Update README.md to contain AMAZON Intent Example

### PR DESCRIPTION
**Pull request by acucciniello**
- I was not sure how to do this to start, until I tried this,
      Someone with less experience developing Skills might not have
      figured it out right away.
    - Meant for Intents  with form of  AMAZON.IntentName
    - It is necessary for people to have some of these intents if they
      are trying to publish their skills.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Changelog
 
 ### 4.0.2 (Next)
-* Your contribution here
```
Comment (dblock) [inline CHANGELOG.md]: You can put this back, and for this we don't need an entry really.

File: README.md [hunk 2]
```diff
@@ -293,6 +293,23 @@ app.intent("vacation", function(request, response) {
...
+
+Amazon has specific intents that have to do with basic functionality of your skill that you must add.  Some examples of this are `AMAZON.HelpIntent`, `AMAZON.StopIntent`, and `AMAZON.CancelIntent`.  Here is an example of how you would specify that type of intent.
+
+```javascript
+app.intent('AMAZON.StopIntent',{
+  'slots': {},
+  'utterances': []
+}, function (request, response) {
+  	var stopOutput = 'Stopping your Request and Exiting Skill'
+  	response.say(stopOutput).send()
```
Comment (dblock) [inline README.md]: This is a good opportunity to be funny. Just saying :) Like `response.say("I'll be back.")`.

File: README.md [hunk 3]
```diff
@@ -293,6 +293,23 @@ app.intent("vacation", function(request, response) {
...
+  	return
+})
+```
+
+You do not need to pass any utterances or slots into these intents.  Also when specifying the name of the intent just use the exact name Amazon gives you.
```
Comment (dblock) [inline README.md]: gives you -> provides

File: README.md (hunk 2 above)
Comment (acucciniello) [inline README.md]: Great idea! I stole your idea and added my own spin

Comment (ericblade) [inline README.md]: Probably should follow the style of the previous example section too, dblock uses double quotes ..

File: README.md (hunk 3 above)
Comment (ericblade) [inline README.md]: May want to provide a link to the page on Amazon about providing utterances for built-in intents, i believe that that is an option ..

**Comment by dblock** [comment]
Merged, thanks.

**Pull request merged**